File: Ad-Hoc/pagina15/2355/2355.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n").map(Number)

let tempo_seg = input.shift()
while (tempo_seg != 0) {
    let ale = Math.ceil(7 * tempo_seg / 90)
    let bra = Math.floor(tempo_seg / 90)

    console.log(`Brasil ${bra} x Alemanha ${ale}`)
    tempo_seg = input.shift()
}

File: Ad-Hoc/__2000__/2355/2355.py
import math
while True:
    tempo_seg = int(input())
    if tempo_seg == 0:
        break
    
    Alemanha_gols = math.ceil(7* tempo_seg / 90)
    Brasil_gols = math.floor(tempo_seg / 90)
    
    print(f"Brasil {Brasil_gols} x Alemanha {Alemanha_gols}")